cli/coverage_cmd.go

```diff
@@ -0,0 +1,179 @@
+package cli
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"go/build"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"golang.org/x/tools/go/loader"
+
+	"github.com/kr/fs"
+
+	"sourcegraph.com/sourcegraph/go-flags"
+)
+
+func init() {
+	cliInit = append(cliInit, func(cli *flags.Command) {
+		_, err := cli.AddCommand("coverage",
+			"use a simple heuristic to check that srclib is outputting expected graph data",
+			`The coverage command acts as a sanity check to ensure that the srclib-go toolchain succeeds when it should and fails when it should not.`,
```

Comment:
Can we not explicitly mention `srclib-go`, since this is the more generic srclib project here.